<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZParsingRulesDLg</class>
 <widget class="QDialog" name="ZParsingRulesDLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source syntax </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZTypeLEd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="SizeFRm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>461</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Header size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HeaderSizeLEd">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Natural size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NaturalLEd">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Universal size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UniversalLEd">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Capacity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CapacityLEd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="KeyEligibleCHk">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key eligible</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateArrayBTn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create array from capacity</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="AcceptBTn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZTypeStrLEd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DiscardBTn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Give up</string>
   </property>
  </widget>
  <widget class="QWidget" name="ZTypeListWDg" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="SyntaxWDg" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
